### ISSUE TITLE
"to html" in Shoes manual is broken

### ISSUE THREAD

**Issue by jasonrclark** [shoes/shoes4#965]
Clicking the "to html" link at the bottom of the manual's left sidebar gives the following crash:

```
NameError: undefined local variable or method `ask_save_folder' for (Shoes::App:0x000038d8 "The Shoes 4 Manual")):Manual
     html_manual at /Users/jclark/source/shoes4/shoes-core/lib/shoes/ui/help.rb:269
        show_toc at /Users/jclark/source/shoes4/shoes-core/lib/shoes/ui/help.rb:101
            call at org/jruby/RubyProc.java:271
      eval_block at /Users/jclark/source/shoes4/shoes-swt/lib/shoes/swt/common/clickable.rb:60
      eval_block at /Users/jclark/.rbenv/versions/jruby-1.7.16.1/lib/ruby/gems/shared/gems/after_do-0.3.1/lib/after_do.rb:99
     handleEvent at /Users/jclark/source/shoes4/shoes-swt/lib/shoes/swt/common/clickable.rb:52
     handleEvent at Shoes$$Swt$$Common$$Clickable$$ClickListener_146472467.gen:13
  run_event_loop at /Users/jclark/.rbenv/versions/jruby-1.7.16.1/lib/ruby/gems/shared/gems/swt-4.4/lib/swt/minimal.rb:109
      event_loop at /Users/jclark/.rbenv/versions/jruby-1.7.16.1/lib/ruby/gems/shared/gems/swt-4.4/lib/swt/minimal.rb:92
            open at /Users/jclark/source/shoes4/shoes-swt/lib/shoes/swt/app.rb:46
        open_gui at /Users/jclark/source/shoes4/shoes-core/lib/shoes/internal_app.rb:103
      initialize at /Users/jclark/source/shoes4/shoes-core/lib/shoes/app.rb:49
             app at /Users/jclark/source/shoes4/shoes-core/lib/shoes/app.rb:19
          (root) at /Users/jclark/source/shoes4/shoes-core/lib/shoes/ui/help.rb:535
            load at org/jruby/RubyKernel.java:1081
          (root) at /Users/jclark/source/shoes4/shoes-core/lib/shoes/ui/manual.rb:1
     show_manual at /Users/jclark/source/shoes4/shoes-core/lib/shoes/ui/manual.rb:4
            load at org/jruby/RubyKernel.java:1081
          (root) at /Users/jclark/source/shoes4/samples/simple-manual.rb:1
          (root) at /Users/jclark/source/shoes4/shoes-core/lib/shoes/ui/cli.rb:1
     execute_app at /Users/jclark/source/shoes4/shoes-core/lib/shoes/ui/cli.rb:41
            load at org/jruby/RubyKernel.java:1081
             run at /Users/jclark/source/shoes4/shoes-core/lib/shoes/ui/cli.rb:55
```

**Comment by jasonrclark** [comment]
Looking over this, I'm actually in favor of just whacking the link instead of trying to perpetuate it. We'll have generated docs online, so I don't see a big reason to keep this around when it's causing issues.

Any dissenting opinions?

**Comment by PragTob** [comment]
@jasonrclark I think the actual idea is that the manual can be used in the app and in the web with the same base. We can whack the link, but the functionality should stay there imo :)

**Comment by jasonrclark** [comment]
@PragTob Totally agreed on the app and web both using the content. Just that the link, and having that logic in the app anyway, is the wrong place for it to live. Will be sure to move the generation itself somewhere more appropriate rather than just deleting.

**Comment by PragTob** [comment]
:+1:

**Comment by ccoupe** [comment]
FWIW, Shoes 3.2 manual doesn't have a 'to html" link although the code still lives in shoes.rb and can be invoked from the command line. I'd be happy to move that out code out of shoes.rb  into deprecated/maual-to-html.rb (example) and kill the entry for the command line. Even if it works on Shoes 3.2 and suspect it doesn't, it's just not relevant any more, IMHO.

**Comment by jasonrclark** [comment]
Slight change of plans to address this from https://github.com/shoes/shoes-manual/pull/1

**Comment by jasonrclark** [comment]
Released 4.0.1 of `shoes-manual` and good to go!
